frontend/src/query_builder/QueryVisualization.jsx

```diff
@@ -188,6 +188,11 @@ export default class QueryVisualization extends Component {
         }
     }
 
+    showDetailError() {
```

Comment:
Using component state for something as simple as this seemed overkill.  If we attempt to use component state then we have a lot more concerns to deal with regarding resetting the state back and forth all the time.  So a simple css show/hide seemed more appropriate.

I can switch the code for accessing the DOM node though.